test/flash_test/src/flash_test.c

```diff
@@ -157,7 +166,7 @@ flash_cli_cmd(int argc, char **argv)
 
         while (off < sz) {
             sec_cnt = min(sizeof(tmp_buf), sz - off);
-            if (hal_flash_write(devid, off, tmp_buf, sec_cnt)) {
+            if (hal_flash_write(fid, off, tmp_buf, sec_cnt)) {
```

Comment:
Why was this updated to `fid`?

Reply:
That's carryover from previous PR attempt, and obviously wrong here. Thanks for spotting it.